homework-submission/abdullahReact/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
What do you need the package.json for?

Reply:
I don't know. I just write "nom init" automatically after NodeJS classes. 
I think I need it if I install React and ReactDOM modules and import right?

Reply:
In this case you are loading React and ReactDOM using script tags in index.html. They are loaded in your browser straight from the url given in the script tag. So your package.json or your modules aren't used at all. No need to run npm init or something the like in this case :-)

Reply:
understand. I just don't used to write code so much. I think didn't get the programming mind yet. but I will soon. thanks for help